src/SeqViz/Circular/CutSites.tsx

```diff
@@ -123,7 +123,7 @@ export default class CutSites extends React.PureComponent<CutSitesProps> {
     if (c.highlightColor) {
       return (
         <CircularFindArc
-          key={c.id}
+          key={JSON.stringify(c)}
```

Comment:
Just curious but what is this for? Won't the ID alone be unique as well?

Reply:
I'm finding that id isn't unique, but `id + recogStrand` is. I'll investigate why

Reply:
turns out there was a key collision between the CutSite and the recogSite